tools/gcsbackup/README.md

```diff
@@ -1,19 +0,0 @@
-# GCSbackup
```

Comment:
My bad, I somehow overlooked that the Bash script was converted to a TS and moved into the main codebase.